contracts/ColonyFunding.sol

```diff
@@ -166,44 +167,38 @@ contract ColonyFunding is ColonyStorage, DSMath {
     return nonRewardPotsTotal[_token];
   }
 
-  function getGlobalRewardPayoutCount() public view returns (uint256) {
-    return globalRewardPayoutCount;
-  }
-
-  function getUserRewardPayoutCount(address _user) public view returns (uint256) {
-    return userRewardPayoutCount[_user];
-  }
-
   function startNextRewardPayout(address _token) public auth {
+    ITokenLocking tokenLocking = ITokenLocking(IColonyNetwork(colonyNetworkAddress).getTokenLocking());
```

Comment:
I guess whoever controls the colony network will be able to change token locking address without colony knowing, so i will change that